samples/bluetooth/mesh/src/microbit.c

```diff
@@ -6,16 +6,27 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <drivers/gpio.h>
-
-#include <display/mb_display.h>
 
 #include <bluetooth/mesh.h>
-
+#include <display/mb_display.h>
+#include <drivers/gpio.h>
 #include "board.h"
 
 static uint32_t oob_number;
 static const struct device *gpio;
+static struct mb_image onoff[] = {
```

Comment:
I realize this is partially a copy-paste, but I think this can be put into ROM by making it `const`